__tests__/sagas-test.ts

```diff
@@ -0,0 +1,18 @@
+import { runSaga } from 'redux-saga';
+import { onGetCounters } from '../src/sagas';
+import { getCounters, getCountersSuccess } from '../src/actions';
+import fetchMock from 'fetch-mock';
+
+it('will fetch counters - success', async () => {
+  const expectedCounters = [{ count: 6 }, { count: 4 }, { count: 2 }, { count: 1 }];
```

Comment:
It doesn't seem that prettier is run on this file. I strongly suggest you setup prettier to run on every file save on your computer. (It should really be a git commit hook, but I don't want to do that just in case it screws up the comet and somehow pushes broken code. But I'm pretty sure I'm just paranoid and it's best to remove any need for discipline in codebases.)